releasenotes/notes/circuit-count-method-a095bd74aaa1d2fb.yaml

```diff
@@ -0,0 +1,27 @@
+---
+features:
```

Comment:
The release notes are user-facing, and we don't need to include information on internal methods and tests. Please edit this down to just the new `job_info()` method and how it works. Something like:

``A new method :meth:`BaseExperiment.job_info` has been added that will output the number of jobs the experiment is expected to be split into based on the provided backend.``

Reply:
okay so i keep this and remove everything else.
```features:
  - |
    Introducing a new method to :class: `BaseExperiment`
    -------------------------------------------------------------------
    A new method :meth:`BaseExperiment.job_info` has been added that will
    output the number of jobs the experiment is expected to be split into
    based on the provided backend.
  - |
    Refer to issue
    `#1247 https://github.com/Qiskit-Extensions/qiskit-experiments/issues/1247`
    for more details.